content/en/synthetics/cicd_testing/ci_results_explorer.md

```diff
@@ -0,0 +1,82 @@
+The CI Results Explorer displays all batches of tests executed via the [Synthetic CI/CD testing integration][1]. Each batch corresponds to a call to the Datadog API (via the NPM package or directly via the API endpoint) to trigger one or several test executions . 
+
+When clicking on a batch, a side panel containing batch CI metadata as well as batch test results shows up, allowing you to look into all test executions that were performed as part of this batch and pinpoint any test failure. You can then click on any failing test result to be redirected to the detailed test result page and look into the root cause of the issue more thoroughly.
+
+## Search
+
+### Facets and tags
+
+The panel on the left side of the page lists several facets you can use to search through your batches:
+
+* **Batch** facets allow you to filter on attributes of your batches:
+
+| Facet            | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| `Summary Status` | The status of the batch: `Passed`, `Failed`, `In Progress`. |
+| `Duration`       | The overall duration of the batch.                          |
+| `ID`             | The id of the batch.                                        |
+
+* **Git** facets allow you to filter on Git related attributes of your batches:
+
+| Facet       | Description                               |
+|-------------|-------------------------------------------|
+| `Branch`    | The branch associated with the batch.     |
+| `Commit SHA`| The commit SHA associated with the batch. |
+
+**CI** facets allow you to filter on CI related attributes of your batches:
+
+| Facet          | Description                                 |
+|----------------|---------------------------------------------|
+| `CI Provider`  | The CI provider associated with the batch.  |
+| `Pipeline URL` | The pipeline URL associated with the batch. |
+
+**Test result** facets allow you to filter on attributes of the test results executed as part of your batches:
+
+| Facet            | Description                                                                                             |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| `Execution Rule` | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, `Skipped`. |
+| `Fast Retries`   | The number of fast retries associated with the test result of the batch.                                |
+| `Location`       | The location associated with the test result of the batch.                                              |
+| `Test ID`        | The test id associated with the test result of the batch.                                               |
+| `Test Name`      | The test name associated with the test result of the batch.                                             |
+
+### Create your query
```

Comment:
```suggestion
### Creating your query
```

Reply:
Same comment as above about keeping the format consistent with https://docs.datadoghq.com/synthetics/search/#create-your-query.